### ISSUE TITLE
Bug Report: Plugin Won't Start.

### ISSUE THREAD

**Issue by JEMcats** [bloomkd46/homebridge-XfinityHome#38]
### Describe The Bug:

I am having an issue where every time the process starts I get an error in the logs and then the plugin restarts.

### To Reproduce:

I don't know what you should do to fix this

### Expected behavior:

the plugin should just start and work like normal

### Logs:

```shell
[12/17/2023, 8:33:17 AM] [Xfinity Home] Restarting Process...
[12/17/2023, 8:33:18 AM] [Xfinity Home] Launched child bridge with PID 8474
[12/17/2023, 8:33:18 AM] Registering platform 'homebridge-xfinityhome.XfinityHomePlatform'
[12/17/2023, 8:33:18 AM] [Xfinity Home] Loaded homebridge-xfinityhome v3.4.2-beta.3 child bridge successfully
[12/17/2023, 8:33:18 AM] [Xfinity Home] Using Refresh Token From Config: **************************************************
[12/17/2023, 8:33:18 AM] [Xfinity Home] Loaded 0 Accessories From Cache
[12/17/2023, 8:33:18 AM] Homebridge v1.7.0 (HAP v0.11.1) (Xfinity Home) is running on port 45991.

/homebridge/node_modules/homebridge-xfinityhome/node_modules/axios/lib/core/createError.js:16
  var error = new Error(message);
              ^
Error: Request failed with status code 400
    at createError (/homebridge/node_modules/homebridge-xfinityhome/node_modules/axios/lib/core/createError.js:16:15)
    at settle (/homebridge/node_modules/homebridge-xfinityhome/node_modules/axios/lib/core/settle.js:17:12)
    at IncomingMessage.handleStreamEnd (/homebridge/node_modules/homebridge-xfinityhome/node_modules/axios/lib/adapters/http.js:322:11)
    at IncomingMessage.emit (node:events:526:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
[12/17/2023, 8:33:18 AM] [Xfinity Home] Child bridge process ended
[12/17/2023, 8:33:18 AM] [Xfinity Home] Process Ended. Code: 1, Signal: null
```

### Config:

```shell
{
            "name": "Xfinity Home",
            "refreshToken": "**************************************************************************",
            "pin": "****",
            "temperatureSensors": true,
            "logLevel": 3,
            "hideUnsupportedDeviceWarnings": false,
            "_bridge": {
                "username": "************",
                "port": *****
            },
            "platform": "XfinityHomePlatform"
        }
```

### Operating System:

Docker

### Node Version:

v20.10.0

### NPM Version:

10.2.3

### Homebridge Version:

1.7.0

### Plugin Version:

v3.4.2-beta.3

**Comment by bloomkd46** [comment]
Can you please try logging into the app on your phone? I had this error come up two weeks ago and it was a result of Xfinity resetting my password. (This was on my alt account). Another week later my primary account needed the password reset. 

Let me know if that doesn't solve the problem

**Comment by home-automate** [comment]
Hi,

Just adding here I too was able to resolve the same issue (also running node 20.10.0 / npm 10.2.3)

I had the same 400 error, though just changing the pw wasn’t enough. Below the sequence I used:
1. disable xfinity home homebridge plugin
2. Update password on Xfinity site
3. From the settings of the plugin in HomeBridge, setup the proxy
4. Login to the xfinity app
5. Confirm token has been captured
6. Uninstall xfinity app
7. Enable xfinity home homebridge plugin
8. Restart homebridge services
9. Everything working again

**Comment by bloomkd46** [comment]
I did just get notified by Xfinity the reason for the password reset. [here](https://assets.xfinity.com/assets/dotcom/learn/Data-Incident.pdf) is the link for those of you who didn’t get the email

**Comment by bloomkd46** [comment]
Fixed v3.4.2
